src/guides/en/understand-devtools/index.md

```diff
@@ -0,0 +1,236 @@
+
+### Differences between browsers
+
+The UI of DevTools is something that each browser implements a little bit differently. The high-level UI description above is a generalization of the UI of DevTools across browsers.
+
+It's a good idea to test your website across different browsers, to find bugs that may manifest on certain browsers only. When this happens, you'll need to use the DevTools of the browser where the bug happens. Therefore, even if this browser isn't the one you normally use, it's good to be familiar with its DevTools.
+
+## The tools
+
+One of the aspects of DevTools that makes it complex is that it contains many different tools. The web development field is vast and there are many different tasks involved with it such as designing the look of a website, writing the code that makes it interactive, optimizing its performance, making sure it's accessible to everyone. DevTools contains tools that help with all of these tasks, and more. On top of this, DevTools accomodates for different types of users as well.
+
+This complexity leads to a high number of tools and features, which translates into a complex UI. But there are two things to keep in mind:
+
+* You don't need to use all the tools all of the time. You can focus on just a few tools that are relevant to the task you're working on.
+* You shouldn't be afraid to explore the tools that you don't know yet. You might find that they become useful to you at certain times.
+
+### The main tools
+
+The tools below tend to be the most used ones. Each tool is listed with a short description of what it's useful for.
+
+| Tool | Description | Use cases | Specific browser note |
+| ---- | ----------- | -------- | --------------------- |
+| **Elements** | Displays the DOM tree and applied CSS rules. | Find and fix HTML and CSS bugs by editing the DOM and CSS styles. | This tool is called **Inspector** in Firefox. |
+| **Console** | JavaScript environment to print messages and evaluate code. | Log messages and debug JavaScript issues. | |
+| **Network** | HTTP network traffic logger. | Improve load performance, debug broken requests, export requests, replay requests. | |
+| **Sources** | JavaScript debugger. | Set breakpoints, step through code, inspect variables. | This tool is called **Debugger** in Firefox. |
+| **Performance** | Performance profiler. | Improve load and runtime performance, find slow JavaScript functions or rendering issues. | |
+| **Memory** | Memory profiler. | Find memory leaks and high memory consumption issues. | |
+| **Application** | Displays local storage, service workers, web app manifests, and other resources. | Debug storage, web applications, and PWA issues. | In Firefox, the **Application** tool only focuses on service workers and web app manifest. Use the **Storage** tool to inspect and edit storage. |
```

Comment:
Also referred as "Storage" in Safari. If we start mentioning these browser-level differences, this could become too verbose.